src/main/java/frc/robot/subsystems/shooter/ShooterConstants.java

```diff
@@ -109,8 +115,44 @@ public static void initConstants() {
                 .withSupplyCurrentLimitEnable(true)
                 .withStatorCurrentLimit(CURRENT_LIMIT_BOTTOM)
                 .withSupplyCurrentLimit(CURRENT_LIMIT_BOTTOM);
+        interpolationMap.putAll(
+                new HashMap<>() {
+                    {
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                        put(new InterpolatingDouble(0), new InterpolatingDouble(0));
+                    }
+                });
```

Comment:
1. `ALL_CAPS`
2. The name is too general.
3. Why declare the constant on line 33 but fill it with values only 85 lines later?
4. It'd be cleaner to save the values in a CSV file and load them here. This will also make the values easier to see in Excel, and will speed up deploy time, because we won't need to rebuild the code just to change some values (only redeploy).

Reply:
b126916
@Emma03L @vichik123 and I agreed a csv a redundent

Reply:
It's very simple to parse a CSV file into a map -- iterate every line, split on a comma, get two numbers, and put them in the map. Much shorter than the current code :-)
It'll also be the same code for both interpolation maps. Data, especially when you have a large quantity of it, should usually be stored in a separate file, which also enables it to be parsed by other software. For the same reason PhotonVision outputs calibration results in a JSON format and not in Java/C++/Python code.

Reply:
I'm actually starting to like this idea. I think I'll implement it.

Reply:
😎